jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1755,6 +1739,15 @@ ecma_builtin_typedarray_prototype_dispatch_routine (uint16_t builtin_routine_id,
     case ECMA_TYPEDARRAY_PROTOTYPE_ROUTINE_REDUCE:
     case ECMA_TYPEDARRAY_PROTOTYPE_ROUTINE_REDUCE_RIGHT:
     {
+      if (info.length == 0)
```

Comment:
This validation should be part of `ecma_builtin_typedarray_prototype_reduce_with_direction `. Instead of passing `arguments_list_p[0]` and `arguments_list_p[1]` directly you can pass `arguments_list_p` and `argument_list_len`

Reply:
This request is still valid.